subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[console]]
+== The Gradle Console
```

Comment:
I see your point. The term "console" could be mistaken or something else.

Which command line chapter are you referring to here?

- https://docs.gradle.org/current/userguide/gradle_command_line.html
- https://docs.gradle.org/current/userguide/tutorial_gradle_command_line.html

Maybe rather the appendix though i'd hate for the command line options to move further down on the page. Now everyone would have to scroll down if you are looking for a command line option.